packages/form/addon/components/cf-field/input/files.js

```diff
@@ -28,8 +29,13 @@ export default class CfFieldInputFilesComponent extends Component {
       },
       "node.value"
     );
-
-    const { downloadUrl } = answers.find((file) => file.name === fileName);
+    const { downloadUrl } =
+      answers.find((file) =>
+        macroCondition(isTesting())
+          ? file.id === fileId ||
+            atob(file.id).substring(file.__typename.length + 1) === fileId
```

Comment:
Why is this needed?

Reply:
Cause the testing graph-ql mock does a base64 encoding of the `{__typename: file.id}` which the real backend does not. I didn't want to mess with the testing setup though.

Reply:
I added a comment in the code